gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+| status | string | One of `scheduled`, `in-progress`, `completed`, or `canceled`. Implementations may prevent certain actions on assignments based on their status at the time (example: an implementation may disallow cancellation of an assigned ride which is `in-progress`).
+
+### Interval Fields
+
+| Name | Type | Description
+|------|--------|------
+| start | datetime | The time at which this interval begins (inclusive)
+| end | datetime | The time at which this interval ends (exclusive)
+
+Intervals determine a discrete and continuous amount of time during which a ride can occur (or more generally, when a thing can occur). Intervals are valid if and only if `start < end`.
+
+### Location Fields
+
+| Name | Type | Description
+|------|--------|------
+| address | [PostalAddress](people.html#postal-addresses) | The physical address for this location
+| identifier | string | A unique identifier within the system for this location. Values may be meaningful to humans ("SC Polling station 1092") or fully opaque database identifiers ("pl2934jk239").
+| availability | Interval[] | If non-empty, availability specifies the times at which this location is available to provide services. For example, a Location representing a polling location might list the hours during which voters are allowed to enter the line to vote.
+
+## Scenarios
+
+The primary motivation for this data model is to describe and accelerate the implementation of applications which provide for matching of drivers and riders, producing assignments. The APIs described here could be used in various ways to build these applications. In one configuration, we can imagine a lower-level routing, dispatch, and scheduling system which is used by API clients to populate their specific rider and driver details, and which provides to those callers a scheduled and on-demand routing and dispatch system. Further, we can imagine that third-parties working as clients to these APIs, but representing neither the specific data owners nor the scheduler's implementation, could build role-specific apps, including mobile apps for drivers or riders, or web-based dispatch and assistance tools for coordinating teams.
+
+While there are any number of ways to factor these various pieces, we present below a small motivating example.
+
+### Scenario: A centrally implemented, web-based GOTV system
+
+In this example, we can imagine a single software team that uses the APIs in this specification to build a self-contained ride coordination service, including:
+  * A single web application and attending database to store the data model above, and which provides services to potential riders, drivers, and campaign coordinators to register, login, and view the system.
+  * Depending on the user's role (driver, rider, administrator), the application presents different views of the data model:
```

Comment:
Currently, I don’t see a role defined for a “Dispatcher,” is that intentional? 
I'd also suggest a "Super Admin" role.

Reply:
It was, since I didn't think it was actually inherent in the data model, and figured that the lower-level types and operations imagined atop them would support any number of roles for users of that system, including "Administrator" and "Dispatcher". The bullets below kind of get at this, but maybe it isn't explicit enough? I'm just trying to keep this model as small as possible, and if things (like dispatcher and admin roles) can be imagined outside of this core model then that was my preference.

Do you want to propose how a formal model would look? Or should we start minimal and add later if necessary?